src/liballoc/boxed.rs

```diff
@@ -283,7 +283,7 @@ impl<T: Clone> Clone for Box<T> {
     /// let x = Box::new(5);
     /// let y = x.clone();
     /// ```
-    #[rustfmt_skip]
+    #[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
`#[rustfmt::skip]` can be used now, fixed by https://github.com/rust-lang/rust/pull/52841